src/components/menu/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { styled } from '@linaria/react'
+import { Text2XS } from '../typography'
+
+export const ElMenuPopover = styled.div`
+  position: absolute;
+  white-space: nowrap;
+  z-index: 98;
+`
+
+export const ElMenu = styled.div`
+  overflow: hidden;
+  display: flex;
+  min-width: 200px;
+  width: fit-content;
+  padding: var(--spacing-2) var(--spacing-none);
+  border-radius: var(--corner-default);
+  background-color: var(--fill-white);
+  box-shadow: 0px 4px 16px 0px #222b3329;
+`
+
+export const ElMenuItemContainer = styled.li`
```

Comment:
>I think this needs some more thought/analysis to work out if there's a simpler way for us to handle this. For example, some questions that come to mind are:
> Do we even need to support the use of a Button component as a child of a MenuItem?

yes, I'm also not sure about that, so I just add a basic style override for `Button`, but I think not supporting it still ok because user still just switch the component and then done.

> Do we need the ElMenuItem to be an <li> element or can it be an <a> or <button> element with a role="menuitem"?

I think `li` will be more flexible because I think the item could also contain `input` probably in the future, if allowed 🤔